lib/v2/secrss/category.js

```diff
@@ -9,14 +10,22 @@ module.exports = async (ctx) => {
     const res = await got(`${api}${category}`);
     const dataArray = res.data.data;
 
-    const items = dataArray.map((item) => ({
-        title: item.title,
-        description: item.summary,
-        pubDate: timezone(parseDate(item.published_at), +8),
-        link: `${host}/articles/${item.id}`,
-        author: item.source_author,
-        category: item.tags.map((t) => t.title),
-    }));
+    const items = await Promise.all(
+        dataArray.map((item) => {
+            const itemUrl = `${host}/articles/${item.id}`;
+            return ctx.cache.tryGet(itemUrl, async () => {
+                const result = await got(itemUrl);
+                const $ = cheerio.load(result.data);
+                const description = $('.article-body').html().trim();
+                return {
+                    title: item.title,
+                    link: itemUrl,
+                    pubDate: timezone(parseDate(item.publishTime), 8),
+                    description,
+                };
```

Comment:
The API does not have `publishTime` field.
`author` and `category` will be gone after this change.